controller/workitem.go

```diff
@@ -72,8 +72,14 @@ func (c *WorkitemController) List(ctx *app.ListWorkitemContext) error {
 		return jsonapi.JSONErrorResponse(ctx, errors.NewBadParameterError("could not parse filter", err))
 	}
 	if ctx.FilterAssignee != nil {
-		exp = criteria.And(exp, criteria.Equals(criteria.Field("system.assignees"), criteria.Literal([]string{*ctx.FilterAssignee})))
-		additionalQuery = append(additionalQuery, "filter[assignee]="+*ctx.FilterAssignee)
+		if *ctx.FilterAssignee == "none" {
```

Comment:
> btw, github doesn't allow this weird names for accounts

@hectorj2f but we will not always having github as our only authn backend, right?